### DISCUSSION TITLE
Problem with return_policy_id field in create draft listing API

### DISCUSSION THREAD

**Discussion by xanhz** [etsy/open-api#854] [Q&A] [answered]
I'm using OpenAPI v3 to create listing on Etsy but I'm got in trouble with field return_policy_id. In one of my stores, when I don't put this field in request body, it throws error like "There was a problem with /return/policy : cannot be null". It's still ok with the other store when this field does not present in request body.
What's problem here? Thanks for all.

**Comment by PMStanley** [comment, marked as answer]
It was covered in January's breaking changes: https://github.com/etsy/open-api/discussions/761

**Reply by xanhz** [reply]
> It was covered in January's breaking changes: https://github.com/etsy/open-api/discussions/761
> 
> 
> 
> 

Thanks for your information
